modules/tensor_mechanics/doc/content/source/utils/RankTwoScalarTools.md

```diff
@@ -40,24 +40,13 @@ where $D$ is the direction vector specified in the input file.
 
 ## Effective Strain
 
-`EffectiveStrain` calculates an effective scalar measure of a Rank-2 tensor, often
-strain ($\epsilon_{ij}$) according to [eq:effective_strain_scalar_type].
-\begin{equation}
-\label{eq:effective_strain_scalar_type}
-s = \sqrt{\frac{2}{3} \epsilon_{ij} \epsilon_{ij}}
-\end{equation}
-
-!alert warning title=This Strain Measure is not for Inelastic Strains
-This effective strain measure, `EffectiveStrain`, should not be confused with
-the effective plastic strain or effective creep strain, which are computed as
-integrals over the history of the inelastic strain; these effective inelastic
-strain measures are computed as scalar material properties (named
-`effective_plastic_strain` and `effective_creep_strain`) for applicable $J_2$
-models. The effective inelastic strains are computed as:
+Effective plastic strain or effective creep strain, which are computed as
+integrals over the history of the inelastic strain as
 \begin{equation}
 s = \int_t\sqrt{\frac{2}{3} \dot{\epsilon}^p_{ij} \dot{\epsilon}^p_{ij}} \mathrm{d}t
```

Comment:
was the EffectiveStrain only for the elastic strain component?  If this changes the effective strain to only be for the inelastic components, maybe this equation should be deleted too since it is no longer used for calculating the effective strain.  Or at least I don't think this is how we calculate the effective creep strain.

Reply:
This equation is how the effective inelastic strain is computed.

Reply:
Or is this equation used for computing the effective strain increment.

Reply:
There is an integral in the equation, so, no, not the increment.

Reply:
I was wondering if we are going to be computing the effective strain increment using this formula 
sqrt{\frac{2}{3} \dot{\epsilon}^p_{ij} \dot{\epsilon}^p_{ij}}
or will we still use the increment computed from the radial return mapping when it is available?   Do we ever reference other theory manuals, like for ls-dyna?  They use the same definition for effective plastic strain

Reply:
If the effective inelastic strain is available from the radial return mapping you should probably use that value, however for stacked models and classes that do not use our radial return mapping you can use this approach.